api/v3/Extension.php

```diff
@@ -386,16 +388,22 @@ function civicrm_api3_extension_getremote($params) {
  * Determine the list of extension keys.
  *
  * @param array $params
+ * @param string $key
  *   API request params with 'keys'.
  *
  * @return array
  */
-function _civicrm_api3_getKeys($params) {
-  if (is_array($params['keys'])) {
-    return $params['keys'];
+function _civicrm_api3_getKeys($params, $key) {
```

Comment:
I suggest giving the default value 'keys' to the 2nd param in this function signature, to better guerantee backward compatibility. Then you won't have to change the function call everywhere and as a side benefit this PR will be smaller/simpler.